### PR TITLE
Makes interface of Laws more consistent by using the Game as a parameter

### LINKED ISSUE #106: Allow check for accepted laws in `Law.effects()`
# Problem

Laws need to be able to check if other laws are accepted - also in the `effects()` function.

E.g. in law [NetzausbauFoerdern](https://github.com/neustartklima/ich-kann-klima/blob/e55ad557c9b8c5c48aa313ee7d138dad05f11f81/src/laws/NetzausbauFoerdern.ts) a check if another law is accepted is needed, but currently not possible, since there is no access to the `game`.

Some TODO comments refering to this issue are marked with #106.

Either:
# Access to game for in Law.effects() to allow use of lawIsAccepted()

That way, laws can check if other laws are accepted as they can in `priority()` now.

# Accepted laws in Parameters

If `data` contained a list of accepted laws, the extension would not be needed.

### PR DESCRIPTION
**Pull request by jschirrmacher**
Uses the game as a parameter for Law.effects() which makes it possible to have a look at already accepted laws in other ones.

This closes #106

### CODE REVIEW

File: tests/Calculator.spec.ts [hunk 1]
```diff
@@ -27,21 +27,21 @@ function mockAcceptedLaw(
     description: id,
     labels,
     treatAfterLabels,
-    effects,
+    effects: effects,
```
Comment (mdrie) [inline tests/Calculator.spec.ts:30]: Is that change needed?

**Review by mdrie — APPROVED** [review]
You are too fast for me!
Great!

File: tests/Calculator.spec.ts (hunk 1 above)
Comment (jschirrmacher) [reply tests/Calculator.spec.ts:30]: Not really. Don't know how it got in there.
